Makefile

```diff
@@ -94,7 +94,11 @@ hack-codegen: hack-build-dev-tools
 
 .PHONY: hack-build
 hack-build:
-	docker build . -t kargo:dev
+	docker build \
+		--build-arg GIT_COMMIT=$(shell git rev-parse HEAD) \
+		--build-arg GIT_TREE_STATE=$(shell if [ -z "`git status --porcelain`" ]; then echo "clean" ; else echo "dirty"; fi) \
+		--tag kargo:dev \
+		.
```

Comment:
This is just a convenience to set build info as accurately as possible if a developer should happen to build from source.